app/src/main/res/values-de/strings.xml

```diff
@@ -192,6 +192,8 @@
     <string name="pref_large_sum">Lade große Bilder wenn verfügbar (z.B. #657). Kann die Performance auf älteren Geräten beeinträchtigen </string>
     <string name="pref_offline">Offline Modus </string>
     <string name="pref_offline_sum">Sichert alle Comics im externen Speicher. Neue Comics werden automatisch hinzugefügt. Belegt ~150MB Speicher</string>
+    <string name="pref_replace_offline">Ersetzen Sie Downloads </string>
+    <string name="pref_replace_offline_sum">Wenn Sie den Offline-Modus aktivieren, laden Sie vorhandene Comic-Dateien herunter und ersetzen Sie sie.</string>
```

Comment:
Should be something like "Bereits vorhandene Comic Dateien ersetzen wenn der Offline Modus aktiviert wird."